www/src/App.tsx

```diff
@@ -17,28 +22,66 @@ const header_sections: Section[] = [
     { title: COMPE_CLUB, url: COMPE_CLUB_ROUTE },
 ];
 
+const StudentApp: FC = () => {
+    return (
+        <Provider store={studentStore}>
+            <Router>
+                <Header sections={header_sections} title={COMPE_PLUS} />
+                <Switch>
+                    <Route path={RESUME_REVIEW_ROUTE}>
+                        <ResumeReview />
+                    </Route>
+                    <Route path={MOCK_INTERVIEW_ROUTE}>
+                        <MockInterview />
+                    </Route>
+                    <Route path={COMMUNITY_ROUTE}>
+                        <Community />
+                    </Route>
+                    <Route path='/'>
+                        <Landing />
+                    </Route>
+                </Switch>
+            </Router>
+        </Provider>
+    );
+};
+
+const VolunteerApp: FC = () => {
+    // TODO: Update components accordingly
```

Comment:
Can you create an issue for this (if one hasn't been made already) pleaseeee

Reply:
Created #138 👍